src/DateUtils/__test__/DateUtils_timezones.spec.js

```diff
@@ -0,0 +1,221 @@
+    ${'US/Eastern'}         | ${'2025-01-01T00:00:00.000Z'} | ${60}  | ${'Tue, 31 Dec 2024 23:00:00 GMT'}
+    ${'Brazil/East'}        | ${'2025-01-01T00:00:00.000Z'} | ${60}  | ${'Tue, 31 Dec 2024 23:00:00 GMT'}
+    ${'UTC'}                | ${'2025-01-01T00:00:00.000Z'} | ${60}  | ${'Tue, 31 Dec 2024 23:00:00 GMT'}
+    ${'Europe/London'}      | ${'2025-01-01T00:00:00.000Z'} | ${60}  | ${'Tue, 31 Dec 2024 23:00:00 GMT'}
+    ${'Australia/Adelaide'} | ${'2025-01-01T00:00:00.000Z'} | ${60}  | ${'Tue, 31 Dec 2024 23:00:00 GMT'}
+    ${'US/Pacific'}         | ${'2025-01-01T00:00:00.000Z'} | ${60}  | ${'Tue, 31 Dec 2024 23:00:00 GMT'}
+  `('supports date from $timezone', ({ timezone, dateStr, offset, expectedUTCString }) => {
+    timezoneMock.register(timezone);
+    let date = new Date(dateStr);
+    date = DateUtils.decreaseDateByOffset(date, offset);
+    expect(date.toUTCString()).toEqual(expectedUTCString);
+  });
+});
+
+describe('increaseDateByOffset', () => {
+  test.each`
+    parameter
+    ${null}
+    ${undefined}
+    ${''}
+    ${'foo'}
+    ${0}
+    ${1}
+    ${{}}
+    ${[]}
+    ${'06/32/2022'}
+  `('should return null if input is "$parameter"', ({ parameter }) => {
+    const mockError = jest.spyOn(console, 'error').mockImplementation(() => {});
+    const res = DateUtils.increaseDateByOffset(parameter, 0);
+    expect(mockError).toHaveBeenCalledTimes(1);
+    expect(res).toBe(null);
+    mockError.mockReset();
+  });
+
+  test.each`
+    timezone                | dateStr                       | offset | expectedUTCString
+    ${'US/Pacific'}         | ${'2025-01-01T00:00:00.000Z'} | ${60}  | ${'Wed, 01 Jan 2025 01:00:00 GMT'}
+    ${'US/Eastern'}         | ${'2025-01-01T00:00:00.000Z'} | ${60}  | ${'Wed, 01 Jan 2025 01:00:00 GMT'}
+    ${'Brazil/East'}        | ${'2025-01-01T00:00:00.000Z'} | ${60}  | ${'Wed, 01 Jan 2025 01:00:00 GMT'}
+    ${'UTC'}                | ${'2025-01-01T00:00:00.000Z'} | ${60}  | ${'Wed, 01 Jan 2025 01:00:00 GMT'}
+    ${'Europe/London'}      | ${'2025-01-01T00:00:00.000Z'} | ${60}  | ${'Wed, 01 Jan 2025 01:00:00 GMT'}
+    ${'Australia/Adelaide'} | ${'2025-01-01T00:00:00.000Z'} | ${60}  | ${'Wed, 01 Jan 2025 01:00:00 GMT'}
+    ${'US/Pacific'}         | ${'2025-01-01T00:00:00.000Z'} | ${60}  | ${'Wed, 01 Jan 2025 01:00:00 GMT'}
+  `('supports date from $timezone', ({ timezone, dateStr, offset, expectedUTCString }) => {
+    timezoneMock.register(timezone);
+    let date = new Date(dateStr);
+    date = DateUtils.increaseDateByOffset(date, offset);
+    expect(date.toUTCString()).toEqual(expectedUTCString);
+  });
+});
+
+describe('addLocalDateToUTCOffset', () => {
+  test.each`
+    parameter
+    ${null}
+    ${undefined}
+    ${''}
+    ${'foo'}
+    ${0}
+    ${1}
+    ${{}}
+    ${[]}
+    ${'06/32/2022'}
+  `('should return null if input is "$parameter"', ({ parameter }) => {
+    const mockError = jest.spyOn(console, 'error').mockImplementation(() => {});
```

Comment:
This mock is used several time. Shouldn't it define outside describe block ?

Reply:
Fixed it :+1: 
Thank you for the feedback